lib/matplotlib/lines.py

```diff
@@ -29,19 +29,28 @@
 _log = logging.getLogger(__name__)
 
 
-def _get_dash_pattern(style):
+def _get_dash_pattern(style, return_ls=False):
```

Comment:
```suggestion
def _get_dash_pattern(style, return_linestyle=False):
```
Even though this is internal, we should document the effect of the new parameter. If you don't tell us about the normalization It's not obvious why the caller wants the linestyle returned when he has just put it in.

Reply:
I have already added that locally.